docs/slides/day2.md

```diff
@@ -0,0 +1,577 @@
+class: middle, center
+
+# JumpStart Live
+In classroom conversation for day 2
+
+---
+
+# Peer code review: [Candy Machine](https://github.com/Ada-Developers-Academy/jump-start/tree/master/learning-to-code/programming-expressions#candy-machine-assignment)
+Pointers for code review:
+
+* Ruby style guide: snake_case, spaces, comments, indentation
+
+* Are all minimal requirements followed and working?
+
+* Are any additional, optional requirements coded and working?
+
+* User experience
+    * Are the prompts useful, educational and sufficient for the user of the
+      program who is unaware of the requirements?
+
+    * What if the user enters information in an incorrect format? (case
+      insensitive input, detailed error messages, prompt for re-entry)
+
+* P.S. For easier code reviews, save files with `.rb` extensions in your gists.
+  This allows for better formatting. Saving multiple `.rb` files helps scope
+  the feedback and test code as you make changes
+
+---
+
+# What will get printed on the terminal?
+
+```ruby
```

Comment:
More of a comment, is there anyway to get Github syntax highlighting for ruby code?

Reply:
There is! There's a range of supported syntax colorschemes for code blocks, including Github. (more info at https://github.com/gnab/remark/wiki/Configuration#highlighting)

You just have to add
```
highlightStyle = "github"
```
to the option parameter when calling `remark.create` in `index.html`

https://github.com/Ada-Developers-Academy/jump-start-live/blob/63d85aff7f368a94a27376dc1e0700f2afb2aa58/docs/index.html#L55-L65

Since all slides use the same styles, we'd just need to do a once-over to make sure nothing got mis-formatted because of the change. It shouldn't, since it's just a color change, but it doesn't hurt to check.

I can make a merge request for it, and then you can preview the change. If it doesn't look good, we can revert / try another color scheme.

Reply:
The style would apply to all code blocks, instead of only to ruby code blocks, but I'm pretty sure that's fine.